### PR TITLE
Fix Time.now format in documentation

### PR DESCRIPTION
**Pull request by tanmay3011**

### CODE REVIEW

File: activesupport/lib/active_support/core_ext/time/conversions.rb [hunk 1]
```diff
@@ -3,28 +3,28 @@
...
-    :iso8601      => lambda { |time| time.iso8601 }
+    iso8601:       lambda { |time| time.iso8601 }
   }
 
   # Converts to a formatted string. See DATE_FORMATS for built-in formats.
   #
   # This method is aliased to <tt>to_s</tt>.
   #
-  #   time = Time.now                    # => Thu Jan 18 06:10:17 CST 2007
+  #   time = Time.now                    # => 2007-01-18 06:10:17 -06:00
```
Comment (zzak) [inline activesupport/lib/active_support/core_ext/time/conversions.rb]: Why did this change?

Comment (tanmay3011) [inline activesupport/lib/active_support/core_ext/time/conversions.rb]: This din't change anything it was just a output correction. The output given earlier was wrong

**Comment by sgrif** [comment]
As I elaborated on in https://github.com/rails/rails/pull/21863#issuecomment-145351491, we don't accept cosmetic patches. The doc changes are fine. The hash syntax is not.

**Comment by tanmay3011** [comment]
@sgrif  Fixed this too!

**Comment by sgrif** [comment]
Thanks. Please add [ci skip] to the commit message for doc changes in the future.

**Pull request merged**